File: sweepai/Calculator.py
def add(num1, num2):
    return num1 + num2

def subtract(num1, num2):
    return num1 - num2

def multiply(num1, num2):
    return num1 * num2

def divide(num1, num2):
    if num2 != 0:
        return num1 / num2
    else:
        print("Error! Division by zero.")
        return None

def main():
    num1 = float(input("Enter the first number: "))
    operator = input("Enter the operator: ")
    num2 = float(input("Enter the second number: "))

    if operator == "+":
        result = add(num1, num2)
    elif operator == "-":
        result = subtract(num1, num2)
    elif operator == "*":
        result = multiply(num1, num2)
    elif operator == "/":
        result = divide(num1, num2)
    else:
        print("Invalid operator.")
        return

    print("The result is: ", result)

if __name__ == "__main__":
    main()

File: csharp/Calculator.cs
using System;

public class Calculator
{
    static void Main(string[] args)
    {
        Console.WriteLine("Enter the first number:");
        double num1 = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("Enter the operator:");
        string op = Console.ReadLine();

        Console.WriteLine("Enter the second number:");
        double num2 = Convert.ToDouble(Console.ReadLine());

        switch (op)
        {
            case "+":
                Console.WriteLine("The result is: " + Add(num1, num2));
                break;
            case "-":
                Console.WriteLine("The result is: " + Subtract(num1, num2));
                break;
            case "*":
                Console.WriteLine("The result is: " + Multiply(num1, num2));
                break;
            case "/":
                if (num2 != 0)
                {
                    Console.WriteLine("The result is: " + Divide(num1, num2));
                }
                else
                {
                    Console.WriteLine("Error! Division by zero.");
                }
                break;
            default:
                Console.WriteLine("Invalid operator.");
                break;
        }
    }

    static double Add(double num1, double num2)
    {
        return num1 + num2;
    }

    static double Subtract(double num1, double num2)
    {
        return num1 - num2;
    }

    static double Multiply(double num1, double num2)
    {
        return num1 * num2;
    }

    static double Divide(double num1, double num2)
    {
        return num1 / num2;
    }
}